java/core/src/java/org/apache/orc/util/CuckooSetBytes.java

```diff
@@ -84,6 +86,9 @@ public CuckooSetBytes(int expectedSize) {
    * and ending at start+len is present in the set.
    */
   public boolean lookup(byte[] b, int start, int len) {
+    if (len < minLen || len > maxLen) {
```

Comment:
In this class, `lookup` seems to be the first method when we call `insert`. Where `maxLen` is initialized before this usage?

Reply:
`maxLen` is declared in the class member variable, so its initial value is 0, and the initial value of `minLen` is `Integer.MAX_VALUE`, so the first element found here basically returns false directly, and then calls insert, which When `minLen` and `maxlen` will be reassigned.